packages/cloud_functions/all_lint_rules.yaml

```diff
@@ -0,0 +1,184 @@
+# Copyright 2021 The Chromium Authors. All rights reserved.
```

Comment:
We could probably move the one in `firebase_core` above `/packages` and reuse it here to avoid duplicates

Reply:
Might be worth getting the individual PRs merged in then doing that since there's a number ongoing at once?

Reply:
In favour of doing this at the end after we've migrated the others, or on a separate PR, wanted to keep the PR self contained to cloud functions changes only.

Tracking internally on INVERTASE/FF-16.